src/parse/asp/config.go

```diff
@@ -82,6 +82,15 @@ func (i *interpreter) newConfig(state *core.BuildState) *pyConfig {
 	base["BUILD_CONFIG"] = pyString(state.Config.Build.Config)
 	base["DEBUG_PORT"] = pyInt(state.DebugPort)
 
+	// Preloaded subincludes don't technically get subincluded so we need to handle them here
+	for _, preloadedSubinclude := range state.Config.Parse.PreloadSubincludes {
+		target := state.Graph.Target(preloadedSubinclude)
+		if target == nil || target.Subrepo == nil {
```

Comment:
These can be nil while we're preloading as we're parsing the packages they're defined in